src/web/routes/register-steps/register-steps.js

```diff
@@ -11,10 +11,10 @@ const { toBooleanStrict } = require('./to-boolean-strict')
  * - toggle must be a string
  * - config must be a boolean (if it exists)
  *
- * N.B. shouldRegisterStep coerces string representations of 'true' and 'false' to boolean
+ * N.B. isStepEnabled coerces string representations of 'true' and 'false' to boolean
  * as it is assumed the config values will be set as environment variables.
  */
-const shouldRegisterStep = config => step => {
+const isStepEnabled = config => step => {
```

Comment:
Renamed this function so that it made sense in terms of being used both by the main application and the acceptance tests.